charts/bookkeeper/templates/bookkeeper.yaml

```diff
@@ -19,29 +19,44 @@ spec:
   probes:
     {{- if .Values.probes.readiness }}
     readinessProbe:
+      {{- if .Values.probes.readiness.initialDelaySeconds }}
       initialDelaySeconds: {{ .Values.probes.readiness.initialDelaySeconds }}
+      {{- end }}
+      {{- if .Values.probes.readiness.periodSeconds }}
       periodSeconds: {{ .Values.probes.readiness.periodSeconds }}
+      {{- end }}
+      {{- if .Values.probes.readiness.failureThreshold }}
       failureThreshold: {{ .Values.probes.readiness.failureThreshold }}
+      {{- end }}
+      {{- if .Values.probes.readiness.successThreshold }}
       successThreshold: {{ .Values.probes.readiness.successThreshold }}
+      {{- end }}
+      {{- if .Values.probes.readiness.timeoutSeconds }}
       timeoutSeconds: {{ .Values.probes.readiness.timeoutSeconds }}
+      {{- end }}
     {{- end }}
     {{- if .Values.probes.liveness }}
     livenessProbe:
+      {{- if .Values.probes.liveness.initialDelaySeconds }}
       initialDelaySeconds: {{ .Values.probes.liveness.initialDelaySeconds }}
```

Comment:
Please add a default value in charts, otherwise if some values are not set by user it wont have any default value

Reply:
done